infra/build/developer-tools/build/scripts/task_helper_functions.sh

```diff
@@ -312,6 +312,17 @@ function generate_modules() {
   fi
 }
 
+# Post a comment to GitHub informing PR author about linting checks status
+# requires a secret with PAT called gh-pat-token and cloud build SA with roles/secretmanager.secretAccessor
+function post_lint_status_pr_comment() {
+  export GITHUB_PAT_TOKEN=$(gcloud secrets versions access latest --secret="gh-pat-token")
+  final_message=$(/usr/local/bin/test_lint.sh --markdown --contrib-guide=../blob/master/CONTRIBUTING.md)
+  if [ -z "$final_message" ]; then
+  final_message="Thanks for the PR! 🚀<br/>✅ Lint checks have passed."
+  fi
+  python3 /usr/local/bin/gh_lint_comment.py -o terraform-google-modules -r "${REPO_NAME}" -p "${_PR_NUMBER}" -c "${final_message}"
```

Comment:
We shouldn't hardcode org, but just make it a default. (Allow overriding via env variable.) Or could this be automatically parsed from the `REPO_NAME` supplied via Cloud Build?

Reply:
I dont [see it being available via CB](https://cloud.google.com/cloud-build/docs/configuring-builds/substitute-variable-values). Did you mean default to `terraform-google-modules` in argparse python script?

Reply:
It should be available as part of either `_HEAD_REPO_URL` or `REPO_NAME` I'd assume?

Reply:
I am using `REPO_NAME` as an input, but only gives the repo_name. I'll check `_HEAD_REPO_URL` which probably has it.

Reply:
Yup so `_HEAD_REPO_URL ` turned out to be in the form `https://github.com/<org>/<repo>`. Made appropriate changes in script to use this instead.

Reply:
Example run: https://github.com/bharathkkb-org/terraform-google-cloud-storage/pull/12

Reply:
@morgante so in case of a fork the `_HEAD_REPO_URL` points to the the fork head branch and not the repo url for the base branch. I could not find another clean way of getting this info using cloud build alone, so does it make sense to just default to `terraform-google-modules` in python script and allow override if needed?

Reply:
That sounds reasonable.

Reply:
Done